english/blast/blast.md

```diff
@@ -0,0 +1,5 @@
+You put your hands together, charge up your energy, and shoot a powerful blast that extinguishes the marshmellow.
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.